round4/team-05.md

```diff
@@ -0,0 +1,162 @@
+
+8. As a traveler with flexible dates, I want the option to sort my flight history by date ranges, so that I can compare prices across different periods and find the cheapest options.
+
+9. As a user who values efficiency, I want the ability to quickly access detailed information on any flight in my history, so that I can make informed decisions without re-searching the flight.
+
+10. As a user who often compares flights, I want the ability to compare selected flights from my history side-by-side, so that I can easily see differences in price, timing, and other relevant details
+ 
+11. As a user with a preference for certain seats, I want to see information on seat availability and type (window, aisle, extra legroom) for flights in my history, so I can make bookings based on my seating preferences.
+
+12. As a user who often compares flights, I want the ability to compare selected flights from my history side-by-side, so that I can easily see differences in price, timing, and other relevant details.
+
+13. As a luxury traveler, I want to filter my flight history to show only business or first-class options, so I can compare premium travel experiences across different airlines.
+
+14. As a parent traveling, I want to filter and sort my flight history based on family-friendly features, such as in-flight entertainment and child meal options, so I can choose the best flights for family travel.
+
+15. As a pet owner, I want to filter my flight history by pet-friendly flights 
+
+## Notes
+
+Potential challenges:
+1. Detecting when to extract data
+2. How will we extract the data from Google Flights.
+
+Features to highlight:
+1. Automatically store information about the flights you’re looking at
+2. Customization and personalization for each user and their needs
+3. Simple and intuitive UI
+
+## References & Inspiration
+
+Other related existing products:
+1. TravelArrow - https://chromewebstore.google.com/detail/travelarrow-your-virtual/coplmfnphahpcknbchcehdikbdieognn
+2. Flight Fare Compare - https://chromewebstore.google.com/detail/flight-fare-compare-googl/mmhdlhieabmpkjcffmaagonallanjiad
+3. CheaperThere - https://chromewebstore.google.com/detail/cheaperthere-cheap-flight/ickdpignpmjcahkjfbmfljhiglmcmchn
+4. Flight Search - https://chromewebstore.google.com/detail/flight-search/dficcobpghplfppapheaenjajahhckkj
+
+
+## Technical Details
+
+### User Interface [melanie]
+
+We plan to have our chrome extension automatically save flight information when the user is on the google flights site. When a user wants to revisit their flight history, there will be a sidebar tab which shows all of their history as well as a filter section. Once the filters are set, the user will then see flights from their history that fall under those filters.
+
+_[Describe which Chrome UI/UX elements you will use in your extension, such as pop-ups, context menus, browser actions, omnibox, sidebar, etc.]_
+
+We will use sidebars.
```

Comment:
I'm curious to know more about what the extension might look like and how users might interact with the various UI elements of the extension